Cargo.toml

```diff
@@ -10,6 +10,7 @@ members = [
   "bottomless",
   "bottomless-cli",
   "libsql-sys-tmp",
+  "libsql-categorizer",
```

Comment:
Can we put this within the libsql crate itself? We could through all the common shared code between sqld and libsql inside the libsql crate.